inc/targetticket.class.php

```diff
@@ -971,6 +973,57 @@ protected function setTargetLocation($data, $formanswer) {
          case self::LOCATION_RULE_SPECIFIC:
             $location = $this->fields['location_question'];
             break;
+         case self::LOCATION_RULE_LAST_ANSWER:
+            $form_id = $formanswer->fields['id'];
+
+            // Get all answers for dropdown questions of this form, ordered
+            // from last to first displayed
+            $answers = $DB->request([
+               'SELECT' => ['answer.plugin_formcreator_questions_id', 'answer.answer', 'question.values'],
+               'FROM' => PluginFormcreatorAnswer::getTable() . ' AS answer',
+               'JOIN' => [
+                  PluginFormcreatorQuestion::getTable() . ' AS question' => [
+                     'ON' => [
+                        'answer' => 'plugin_formcreator_questions_id',
+                        'question' => 'id',
+                     ]
+                  ]
+               ],
+               'WHERE' => [
+                  'answer.plugin_formcreator_formanswers_id' => $form_id,
+                  'question.fieldtype'                       => "dropdown",
```

Comment:
```suggestion
                  'question.fieldtype'                       => 'dropdown',
```

Reply:
Not sure about this one, GLPI coding standards allow both syntaxes.
I think it's best to use the same "core" coding standards for all plugins or we'll end up with 10 different standards very quickly 😅

Reply:
GLPI coding standards allow both syntaxes, then using single quotes is better as there is a very slight performance gain. Repeated by thousands of strings, there is probably a measurable benefit. Green coding.